lib/experimental/Navigation/Page/Layouts/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { FullWidthLayout } from "./FullWidthLayout"
+import { StandardLayout } from "./StandardLayout"
+import { LayoutType } from "./types"
+
+export { FullWidthLayout, StandardLayout }
+
+interface LayoutProps {
+  layout: LayoutType
+  children: React.ReactNode
+}
+
+export function Layout({ layout, children }: LayoutProps) {
+  switch (layout) {
+    case "fullWidth":
+      return <FullWidthLayout>{children}</FullWidthLayout>
+    case "standard":
+    default:
+      return <StandardLayout>{children}</StandardLayout>
+  }
+}
```

Comment:
At this point, why not just allowing people to use `<FullWidthLayout>` directly under the `Page` children? I understand this way you force *something* to be there, but at the same time you limit props. I can imagine a near future where we add an `InboxLayout` where we want to pass a prop for `list` and another one for `detail`. This API design would make it quite difficult.